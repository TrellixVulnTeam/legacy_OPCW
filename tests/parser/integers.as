# Decimal
235627;
32;
4930782690;

# Binary
0b10100101010101;
0B0011101;

# Octal
0o755;
0O644;
0o777;

# Hexadecimal
0xDeadBeef;
0XABCDEF;
0xff;
0x0b;
0x0A;
