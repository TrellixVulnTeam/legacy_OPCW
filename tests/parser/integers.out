ModuleDecl <?>
  Nodes <?> 
    IntegerExpr <?> 'int' 235627 (10)
    IntegerExpr <?> 'int' 32 (10)
    IntegerExpr <?> 'int' 4930782690 (10)
    IntegerExpr <?> 'int' 10100101010101 (2)
    IntegerExpr <?> 'int' 0011101 (2)
    IntegerExpr <?> 'int' 755 (8)
    IntegerExpr <?> 'int' 644 (8)
    IntegerExpr <?> 'int' 777 (8)
    IntegerExpr <?> 'int' DeadBeef (16)
    IntegerExpr <?> 'int' ABCDEF (16)
    IntegerExpr <?> 'int' ff (16)
    IntegerExpr <?> 'int' 0b (16)
    IntegerExpr <?> 'int' 0A (16)
